libstuff/libstuff.cpp

```diff
@@ -2522,15 +2505,98 @@ int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int6
     uint64_t startTime = STimeNow();
     int error = 0;
     int extErr = 0;
+
+    size_t numLoops = 0;
+    size_t prepareTimeUS = 0;
+    size_t numSteps = 0;
+    size_t stepTimeUS = 0;
+    size_t longestStepTimeUS = 0;
```

Comment:
Add a bunch of timing for the sub-parts of running a query to diagnose where we might get stuck with sqlite.